### ISSUE TITLE
I have a question. 

### ISSUE THREAD

**Issue by kwbyun** [realgump/MvMHAT#15]
Thanks for revealing a great project.

I downloaded and trained the pre-trained model of resnet-50.

However, unlike the result of 1-1, in the case of 1-2, the instance id starts from 6.

I set the epoch to around 30.

By any chance, can learning be performed properly only when the initial setting is about 1000 epochs?

The loss decreased from 0.37 to 0.23, but I wonder how much the loss must be reduced for proper performance.

thank you
